### PR TITLE
Feature/dialog component

### PR DESCRIPTION
**Pull request by adnan1naeem**
## What I did
DIalog Component
## How to test
no
### Is this testable with jest or storyshots?
no
### Are any of the existing tests are failing ?
no
### Does it have breaking changes ?
no
If your answer is yes to any of these, please make sure to include it in your PR.

@bjavaid @amnajunaid please review the changes and merge them.

### CODE REVIEW

File: src/components/Dialog/Dialog Story.tsx [hunk 1]
```diff
@@ -0,0 +1,93 @@
+import React, { Component, ReactNode } from 'react';
```
Comment (amnajunaid) [inline src/components/Dialog/Dialog Story.tsx]: @adnan1naeem please see following comments.
- Remove unused Component import.
- Rename file name to exclude space.

File: src/components/Dialog/Dialog Story.tsx [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+import { Button } from '../Button';
+import { ModalProperties } from '@blueeast/bluerain-ui-interfaces';
```
Comment (amnajunaid) [inline src/components/Dialog/Dialog Story.tsx]: @adnan1naeem  Where ModalProperties is being used ?

File: src/components/Dialog/Dialog Story.tsx [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+import  Dialog from   './Dialog.Component';
+import DialogBody from  './DialogBody.Component';
+import   DialogFooter from './DialogFooter.Component';
```
Comment (amnajunaid) [inline src/components/Dialog/Dialog Story.tsx]: Please remove extra spaces from imports.

Comment (adnan1naeem) [reply src/components/Dialog/Dialog Story.tsx]: fixed

File: src/components/Dialog/Dialog Story.tsx (hunk 2 above)
Comment (adnan1naeem) [reply src/components/Dialog/Dialog Story.tsx]: fixed

File: src/components/Modal/Modal.Component.tsx [hunk 4]
```diff
@@ -0,0 +1,52 @@
+
```
Comment (amnajunaid) [inline src/components/Modal/Modal.Component.tsx]: @adnan1naeem  
- Make comoponents stateless, no need to make classes.
- Props should be passed through spread operator.
- Remove unused Button, ReactNode, Component imports from all files
- Remove actions, knobs unused imports from .stories files.

**Pull request merged**